README.md

```diff
@@ -38,20 +38,13 @@ All commands:
 
 1. Configure [direv](https://direnv.net/). Then copy `.sample.envrc` to a new `.envrc`. Don't forget to run `direnv allow` after any changes to `.envrc`.
 
-2. Install [docker](https://www.docker.com/get-started/) onto your machine, if you do not have it. Execute `docker-compose up -d` in the root of the repo. This starts the database. NOTE: If you've never run this command before, this will build a new image of the postgres database.
+2. Install [docker](https://www.docker.com/get-started/) onto your machine, if you do not have it. Execute `docker-compose up` in the root of the repo. This starts the database and pheonix servers.
 
-3. Install the tool versions necessary for this repo. If you are using [asdf](https://asdf-vm.com/), run these commands to install everything in the `.tool-versions` file:
-  * `asdf plugin add elixir`
-  * `asdf install`
+3. Create and migrate your database with the command: `docker compose run app mix ecto.migrate `.
```

Comment:
When I was running through this, I found that I have to execute a step before this. I had to install the mix dependencies with `docker compose run app mix deps.get`. Then I was able to run migrations.